docs/_extra/api-reference/hypothesis-v1.yaml

```diff
@@ -14,29 +14,17 @@ info:
     # Hypothesis API
 
     This is a reference for the Hypothesis HTTP API version 1.0.
-    See [the API overview](../api/) for an introduction to the API and information
-    on how authorization works.
-
-    ## Expanding Resources
-
-    Resources often contain links to related resources in their response bodies.
-    Some endpoints support resource expansion, allowing those related resources
-    to be expanded inline within the response.
 
+    See [the API overview](https://h.readthedocs.io/en/latest/api/)
```

Comment:
Was there a particular reason for changing to an absolute URI here? I presume this is copied from the `hypothesis-v2.yaml` docs?

Reply:
Yes! This OpenAPI document is standalone—it should work outside of the context of our docs site.